playbooks/installation/index.adoc

```diff
@@ -95,6 +95,11 @@ Each service in the cluster will be assigned an IP from this range.  The default
 
 Both the openshift_portal_net and the osm_cluster_network_cidr will default to the value above if not set, however if you prefer to have your inventory file describe your environment, its recommended to set them explicitly for future reference or comparison.
 
+==== Host Subnet Length
+*Ansible Host File Variable:* `osm_host_subnet_length=9`
+
+Each OpenShift node will require its own subnet which allows the pods to obtain an IP in that range.  This variable in the ansible hosts file sets that subnet.  The 9 in the example above represents a /23 subnet or 2 class C subnets or also described as 510 usable IP addresses.  What does this have to do with the /14 cluster network mentioned above.  It all boils down to a network math problem.  We have a total number of IP addresses to use that is established by the /14 refereced above.  So now to the math:  262,142 (total IP) / 510 (total IP per node) = ~514 total nodes in your cluster that will run a total of 510 pods on each node in your cluster.
```

Comment:
I think the math equiatiot should break out into its own paragraph, and be bolded and italicized.